docs/analyzer-rules/AZFW0010.md

```diff
@@ -0,0 +1,24 @@
+# AZFW0010: Invalid binding type
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW00010|
+| **Category** |[Usage]|
+| **Severity** |Warning|
+
+## Cause
+
+This rule is triggered when a function is binding to a type that is not supported
+by the binding attribute being used.
+
+## Rule description
+
+Some bindings advertise the types that they support. If a binding advertises supported types
+and does not support falling back to built-in converters, then this rule will will be flagged.
+
+For example if you're binding to a `QueueTrigger` and the only supported type is `string` and
+your function is binding to a `bool`, then you will see this warning.
+
+## How to fix violations
+
+Change the binding type to a type that is supported by the binding your function is using.
```

Comment:
Should we also list the types supported by each input bindings here to help the user? I think that might be helpful for the user.